src/js/components/Navigation/SettingsPersonalSideBar.jsx

```diff
@@ -72,8 +72,17 @@ export default class SettingsPersonalSideBar extends Component {
   }
 
   voterSignOut = () => {
+    // GTM dataLayer for Sign Out
+    TagManager.dataLayer({
+      dataLayer: {
```

Comment:
Please make sure you have the latest code, and then look at the dataLayer around line 190 of this file to get a feel for all of the other dictionaries you will need to add here: src/js/components/Share/ShareModal.jsx
You will need to add more to this dataLayer, including actionDetails, pageDetails, and userDetails.
Instead of introducing "settings_section", I would add:
```
actionDetails: {
    actionType: 'signOut',
    buttonId: 'signOutPersonalSidebar',
}
```
...then I would add `id="signOutPersonalSidebar"` to "BorderTopContainer"